src/catalog/inventory-configure-source-priority.md

```diff
@@ -16,8 +16,23 @@ When run, the algorithm:
 
 To configure, arrange those sources from top-to-bottom in priority for fulfilling orders. The Source Selection Algorithm (SSA) provides an algorithm Priority using this order when determining shipment and inventory deductions. See [Prioritizing Sources for a Stock]({% link catalog/inventory-stock-priority.md %}).
 
+## Configure the priority of sources
+
+1. On the _Admin_ sidebar, go to **Stores** > **Inventory** > **Stocks**.
+
+1. Open a stock in **Edit** mode and scroll down the page to the _Sources_ area.
+
+1. Click **Assign Sources**.
+
+    This displays the **Assign Sources page**. Select the checkbox for the required source and click <span class="btn">Done</span> to assign a source to the stock.
```

Comment:
This displays the **Assign Sources** view. Select the checkbox for the required source and click **Done** to assign a source to the stock.

Reply:
Thanks, Done ✅ 
Please, approve.